# # Definitions of tags to detect
#
# ## General remarks
#
# - All length in meters
# - Ellipsis (...) signifies that the previous element can be repeated multiple times.
#
# ## Standalone tag definitions
# ### Remarks
#
# - name is optional
#
# ### Syntax
#
# standalone_tags:
#   [
#     {id: ID, size: SIZE, name: NAME},
#     ...
#   ]
standalone_tags:
  [
    {id: 58, size: 0.065, name: TagAA1},
    {id: 20, size: 0.065, name: TagAA1},
  ]

tag_bundles:
  [
  ]

  # # Definition of tag positions and orientations, defined as the world frame pose in the tag frame.
#standalone_tag_poses:
#  [
#
#    {id: 20, x: 0.3075, y: -0.0725, z: 0.585, qx: -0.5, qy: 0.5, qz: 0.5, qw: 0.5},
#    {id: 16, x: -0.3575, y: -0.0725, z: 0.585, qx: -0.5, qy: 0.5, qz: 0.5, qw: 0.5}

#  ]

intersection_tuples:
  [
  ]
